src/main/java/no/digipost/monitoring/micrometer/ApplicationInfoMetrics.java

```diff
@@ -79,4 +76,13 @@ public void bindTo(MeterRegistry registry) {
             .tags(tags)
             .register(registry);
     }
+
+    private String fromManifestOrEnv(String name){
+        String value = manifest.getMainAttributes().getValue(name);
+        if(value == null) {
+            value = System.getenv(name);
+        }
+        Objects.requireNonNull(value, name + " was null. Set it in MANIFEST.MF or as an environment variable");
```

Comment:
This changes behavior. What if it is ok that neither of these are set? (Like in tests)